keyboards/alpha/keymaps/default/keymap.c

```diff
@@ -0,0 +1,88 @@
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
Missing the keyboard include now (`#include QMK_KEYBOARD_H`)

Reply:
That's really odd, it's on my file. I'm re-pushing.